eol_contact_form/locale/en/LC_MESSAGES/django.po

```diff
@@ -80,85 +70,173 @@ msgid "Contact Form"
 msgstr ""
 
-#: eol_contact_form/templates/eol_contact_form/contact.html:62
-msgid "Select an option"
+#: eol_contact_form/templates/eol_contact_form/contact.html:50
+msgid "Example: xxxxxxx@xxxx.xxx"
+msgstr ""
+
+#: eol_contact_form/templates/eol_contact_form/contact.html:53
+msgid "Please tell us which category your question belongs to"
+msgstr ""
+
+#: eol_contact_form/templates/eol_contact_form/contact.html:55
+msgid "Select a category"
+msgstr ""
+
+#: eol_contact_form/templates/eol_contact_form/contact.html:56
+msgid "Specific course questions"
+msgstr ""
+
+#: eol_contact_form/templates/eol_contact_form/contact.html:57
+msgid "Issuance of Certificates"
+msgstr ""
+
+#: eol_contact_form/templates/eol_contact_form/contact.html:58
+msgid "Where to find the course programs"
+msgstr ""
+
+#: eol_contact_form/templates/eol_contact_form/contact.html:59
+msgid "Problems in the progress"
+msgstr ""
+
+#: eol_contact_form/templates/eol_contact_form/contact.html:60
+msgid "Content of a course"
 msgstr ""
 
-#: eol_contact_form/templates/eol_contact_form/contact.html:63
-msgid "General"
+#: eol_contact_form/templates/eol_contact_form/contact.html:61
+msgid "Administrative"
+msgstr ""
+
+#: eol_contact_form/templates/eol_contact_form/contact.html:62
+msgid "Evaluations"
 msgstr ""
 
 #: eol_contact_form/templates/eol_contact_form/contact.html:64
-msgid "Specific Course"
+msgid "Technical problems"
 msgstr ""
 
 #: eol_contact_form/templates/eol_contact_form/contact.html:65
-msgid "Login"
+msgid "Login problems"
 msgstr ""
 
 #: eol_contact_form/templates/eol_contact_form/contact.html:66
-msgid "Other"
+msgid "Identity verification"
+msgstr ""
+
+#: eol_contact_form/templates/eol_contact_form/contact.html:67
+msgid "Platform errors"
+msgstr ""
+
+#: eol_contact_form/templates/eol_contact_form/contact.html:69
+msgid "Use of the platform"
 msgstr ""
 
 #: eol_contact_form/templates/eol_contact_form/contact.html:70
-msgid "Course name"
+msgid "Creation of courses"
 msgstr ""
 
 #: eol_contact_form/templates/eol_contact_form/contact.html:71
-msgid "Enter the course name"
+msgid "Teacher training"
+msgstr ""
+
+#: eol_contact_form/templates/eol_contact_form/contact.html:72
+msgid "Honor code"
 msgstr ""
 
 #: eol_contact_form/templates/eol_contact_form/contact.html:74
-msgid "Describe your problem"
+msgid "Other"
 msgstr ""
 
 #: eol_contact_form/templates/eol_contact_form/contact.html:75
-msgid "Enter your questions and queries here"
+msgid "Congratulations"
+msgstr ""
+
+#: eol_contact_form/templates/eol_contact_form/contact.html:76
+msgid "Comments or suggestions"
+msgstr ""
+
+#: eol_contact_form/templates/eol_contact_form/contact.html:77
+msgid "Complaints"
+msgstr ""
+
+#: eol_contact_form/templates/eol_contact_form/contact.html:82
+msgid "Please tell us the name of the course"
+msgstr ""
+
+#: eol_contact_form/templates/eol_contact_form/contact.html:83
+msgid "Example: Course name"
```

Comment:
Agregar un ejemplo que tenga sentido y sirva para entender el concepto, p.e. "Example: Introduction to EOL platform" o similar.